eng/helix/content/InstallJdk.ps1

```diff
@@ -0,0 +1,62 @@
+<#
```

Comment:
Can the existing `InstallJdk` scripts from eng/scripts/ grow parameters to avoid duplicating them?

Reply:
Unfortunately we need the script to be in this folder for helix, so either we duplicate the script, or use this one for everything